README.md

```diff
@@ -1,5 +1,79 @@
 # mhc-fe
-my health connect frontend repository with design studio and frontend source code
+<h1>my health connect frontend repository with design studio and frontend source code</h1>
 
+Design tokens are the visual design atoms of the design system — specifically, they are named entities that store visual design attributes, we can say they are variables that can be given a value.
```

Comment:
add the following before your description.

"### How to integrate figma, figma token to this repository"